Overview.bs

```diff
@@ -342,23 +343,133 @@ response_type can be one of the following values:
+    client is capable of decoding. At least one format must be provided.
+
+### GET ### {#rebase-request-get}
+
+If sent as a GET request the client will include a single query parameter:
+
+*  <code>request</code>:<br/>
+    The value is a single <a href="#PatchRequest"><code>PatchRequest</code></a> object encoded via CBOR
+    and then base64url encoding [[rfc4648]]. The <a href="#PatchRequest"><code>PatchRequest</code></a>
+    object uses the same fields as specified for a <a href="#rebase-request-post">POST</a> request.
+
 ### Standard Response ### {#rebase-standard-response}
 
+The server will response to a new font request with a <a href="#rebase-response">New Font Response</a>.
+
 ### Errors ### {#rebase-errors}
 
+The following errors may occur as a result of a new font request:
+
+*  [[#error-font-not-found]]
+*  [[#error-bad-request]]
+*  [[#error-internal-error]]
+
 Patch Font Request {#patch-request}
 -----------------------------------
 
+A patch font request is sent by a client to add data for additional codepoints to its font. Patch font
+requests are sent via HTTP and can only use the POST method. The request body is a single
+<a href="#PatchRequest"><code>PatchRequest</code></a> object encoded via CBOR. The fields of
+should be set as follows:
+
+*  <a href="#PatchRequest"><code>PatchRequest.original_font_checksum</code></a>:<br/>
+    Set to the last saved value of original_font_checksum provided by a
+    previous response from the server for this particular font.
+   
+*  <a href="#PatchRequest"><code>PatchRequest.base_checksum</code></a>:<br/>
+    Set to the checksum [[#computing-checksums]] of the client’s most recent
+    copy of the font.
+   
+*  <a href="#PatchRequest"><code>PatchRequest.patch_format</code></a>:<br/>
+    This field should be populated with the set of <a href="#patch-formats">patch formats</a> that this
+    client is capable of decoding. At least one format must be provided.
+
+If the server has previously provided a <code>codepoint_ordering</code> for this
+font the client should set:
+
+*  <a href="#PatchRequest"><code>PatchRequest.ordering_checksum</code></a>:<br/>
+    Set to the most recent value of <code>ordering_checksum</code> provided by a
+    response from the server for this particular font.
+   
+*  <a href="#PatchRequest"><code>PatchRequest.indices_have</code></a>:<br/>
+    The set of codepoints that are covered by the client’s current copy of the font.
+    Specified using indices obtained by applying <code>codepoint ordering</code>
+    [[#codepoint-reordering]] to the set of codepoints.
+   
+*  <a href="#PatchRequest"><code>PatchRequest.indices_needed</code></a>:<br/>
+    The set of codepoints that the clients would like added to it’s copy of
+    the font. Specified using indices obtained by applying <code>codepoint ordering</code>
+    [[#codepoint-reordering]] to the set of codepoints.
+
```

Comment:
I suggest that we should maintain a clear differentiation between _codepoints_ and _indices_, and not use these terms interchangeably. Indices are commonly associated with glyph IDs while codepoints are units of text content that, depending on a font used, can be associated with more than one glyph ID (contextual forms, stylistic sets, etc.).

Reply:
I reworded this slightly to try and make it more clear.